server/huntflow_reloaded/tokens.py

```diff
@@ -0,0 +1,99 @@
+""" JWT module """
+import os
+import sys
+from datetime import datetime, timedelta
+from uuid import uuid4
+
+import jwt
+
+if sys.argv[0].split('/')[-1] == 'testing.py':
+    os.environ['ACCESS_TOKEN_LIFETIME'] = '0.5'
+    os.environ['REFRESH_TOKEN_LIFETIME'] = '1'
+    os.environ['SECRET_KEY'] = 'secret'
+
+
+class ExpiredTokenException(Exception):
+    """Exception raised when token is expired. """
+
+class Token():
+    """Abstract class for the token instance. """
+
+    token_type = None
+    lifetime = None
+
+    def __init__(self, token=None):
+        self.token = token
+        self.current_time = datetime.now()
+
+        if token is not None:
+            self.decode(token)
+        else:
+            self.payload = {'token-type': self.token_type}
+            self.payload['jti'] = uuid4().hex
+            self.payload['exp'] = self.current_time + timedelta(
+                minutes=float(self.lifetime))
+
+    def decode(self, token):
+        """Decodes the given token. """
+
+        try:
+            secret = os.getenv('SECRET_KEY') or 'secret'
```

Comment:
1. `os.getenv('SECRET_KEY', 'secret')`
2. Move `'secret'` to something like `DEFAULT_SECRET_KEY`